### PR TITLE
 Closes #251 - Create Unit Tests (Part 3)

### LINKED ISSUE #251: Create Unit Tests (Part 3)
CourseController
DeckCommentController
DeckController

### PR DESCRIPTION
**Pull request by maxturko**
Created unit tests for all working methods in:

DeckController
DeckCommentController
CourseController

Fixed method addDeckForUser() in DeckController.java.

Added setters to ReplyToCommentDTO.java for unit tests.

### CODE REVIEW

**Review by antonbakalets — APPROVED** [review]

File: src/main/java/com/softserve/academy/spaced/repetition/controller/DeckController.java [hunk 1]
```diff
@@ -111,9 +111,13 @@ public DeckPublicDTO getDeckById(@PathVariable Long deckId) {
...
                 .getCardsByCourseAndDeck(categoryId, courseId, deckId)).withSelfRel());
     }
+    /*  TODO method addDeckToCategory had same PostMapping value as addDeckForUser()
+     *  temporarily changed categories -> categories1 in value so other
+     *  method can work
+     */
 
     @Auditable(action = AuditingAction.CREATE_DECK_IN_CATEGORY)
-    @PostMapping(value = "/api/categories/{category_id}/decks")
+    @PostMapping(value = "/api/categories1/{category_id}/decks")
```
Comment (Maks-Chornyi) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/DeckController.java:120]: why did you add "1" to this url?

Comment (maxturko) [reply src/main/java/com/softserve/academy/spaced/repetition/controller/DeckController.java:120]: It is a hotfix to a problem where two controllers had same Postmapping value
method addDeckToCategory had same PostMapping value as addDeckForUser()
temporarily changed categories -> categories1 in value so other
method can work

**Review by Maks-Chornyi — APPROVED** [review]

**Review by OlehPaliuh — APPROVED** [review]

**Review by OleksandrBartosh — APPROVED** [review]
